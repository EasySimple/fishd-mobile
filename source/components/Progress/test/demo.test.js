import webDemoTest from '../../../tests/demoTest';

webDemoTest('Progress');
