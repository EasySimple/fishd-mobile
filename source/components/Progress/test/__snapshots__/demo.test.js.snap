// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Progress demo correctly 1`] = `
<div>
  <div
    class="components-tpl-demo-basic"
  >
    <div
      class="demo-title"
    >
      基础用法
    </div>
    <div
      class="demo-card"
    >
      <div
        class="fm-progress"
        style="background:#e5e5e5;height:4px"
      >
        <span
          class="fm-progress__portion"
          style="background:#1989fa;width:0px"
        >
          <span
            class="fm-progress__pivot"
            style="color:#fff;left:0px;background:#1989fa"
          >
            50%
          </span>
        </span>
      </div>
    </div>
  </div>
  <div
    class="components-tpl-demo-basic"
  >
    <div
      class="demo-title"
    >
      线条粗细
    </div>
    <div
      class="demo-card"
    >
      <div
        class="fm-progress"
        style="background:#e5e5e5;height:8px"
      >
        <span
          class="fm-progress__portion"
          style="background:#1989fa;width:0px"
        >
          <span
            class="fm-progress__pivot"
            style="color:#fff;left:0px;background:#1989fa"
          >
            50%
          </span>
        </span>
      </div>
    </div>
  </div>
  <div
    class="components-tpl-demo-basic"
  >
    <div
      class="demo-title"
    >
      置灰
    </div>
    <div
      class="demo-card"
    >
      <div
        class="fm-progress"
        style="background:#e5e5e5;height:4px"
      >
        <span
          class="fm-progress__portion"
          style="background:#cacaca;width:0px"
        >
          <span
            class="fm-progress__pivot"
            style="color:#fff;left:0px;background:#cacaca"
          >
            50%
          </span>
        </span>
      </div>
    </div>
  </div>
  <div
    class="components-tpl-demo-basic"
  >
    <div
      class="demo-title"
    >
      样式定制
    </div>
    <div
      class="demo-card"
    >
      <div
        class="fm-progress"
        style="background:#e5e5e5;height:4px"
      >
        <span
          class="fm-progress__portion"
          style="background:#f5222d;width:0px"
        >
          <span
            class="fm-progress__pivot"
            style="color:#fff;left:0px;background:#f5222d"
          >
            红色
          </span>
        </span>
      </div>
      <div
        class="fm-progress"
        style="background:#e5e5e5;height:4px"
      >
        <span
          class="fm-progress__portion"
          style="background:#52c41a;width:0px"
        >
          <span
            class="fm-progress__pivot"
            style="color:#fff;left:0px;background:#52c41a"
          >
            绿色
          </span>
        </span>
      </div>
      <div
        class="fm-progress"
        style="background:#e5e5e5;height:4px"
      >
        <span
          class="fm-progress__portion"
          style="background:#722ed1;width:0px"
        >
          <span
            class="fm-progress__pivot"
            style="color:#fff;left:0px;background:#722ed1"
          >
            紫色
          </span>
        </span>
      </div>
    </div>
  </div>
</div>
`;
